habitat-hitl/habitat_hitl/app_states/app_service.py

```diff
@@ -4,28 +4,43 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from __future__ import annotations
+
+from typing import Callable, Optional
+
+from habitat import Env
+from habitat.tasks.rearrange.rearrange_sim import RearrangeSim
+from habitat_hitl.core.client_message_manager import ClientMessageManager
+from habitat_hitl.core.gui_input import GuiInput
+from habitat_hitl.core.remote_gui_input import RemoteGuiInput
+from habitat_hitl.core.serialize_utils import BaseRecorder
+from habitat_hitl.core.text_drawer import AbstractTextDrawer
+from habitat_hitl.environment.controllers.controller_abc import GuiController
+from habitat_hitl.environment.episode_helper import EpisodeHelper
+from habitat_sim.gfx import DebugLineRender
+
 
 # Helpers to provide to AppState classes, provided by the underlying SandboxDriver
 class AppService:
     def __init__(
         self,
         config,
```

Comment:
May want to do this to force the arguments to be always keywords. The docs generator will pick that up and show in the output as well.

```suggestion
        *, config,
```

Reply:
Neat trick! Thank you!